docs/api/profiles.md

```diff
@@ -0,0 +1,86 @@
+# Profiles
+
+Profiles contain information about a User. A User does not need a Profile, but a Profile is always created
+together with a User.
+
+## Get the profile of a user
+
+### URL
+
+```http
+GET /profiles/:profile HTTP/1.1
```

Comment:
Both actually, currently the current user thing is good enough. I've also set up basicauths like that.